edx_event_bus_kafka/internal/consumer.py

```diff
@@ -88,6 +84,7 @@ def inner_from_dict(event_data_dict, ctx=None):  # pylint: disable=unused-argume
             'value.deserializer': AvroDeserializer(schema_str=signal_deserializer.schema_string(),
                                                    schema_registry_client=schema_registry_client,
                                                    from_dict=inner_from_dict),
+            # TODO: Possibly remove or change the offset behavior https://github.com/openedx/event-bus-kafka/issues/61
```

Comment:
We already switched to latest. I don't think we need a todo here. I added a comment to the issue to get it closed.

Reply:
Oops, thanks, missed that in the rebase.